cpython/Makefile

```diff
@@ -102,7 +102,7 @@ $(BUILD)/Makefile: $(BUILD)/.patched $(ZLIBBUILD)/.configured $(SQLITEBUILD)/lib
 	cp config.site $(BUILD)/
 	( \
 		cd $(BUILD); \
-		CONFIG_SITE=./config.site READELF=true LD_RUN_PATH="$(SQLITEBUILD):$(BZIP2BUILD)" emconfigure \
+		EMCC_CFLAGS="-s WASM=0" CONFIG_SITE=./config.site READELF=true LD_RUN_PATH="$(SQLITEBUILD):$(BZIP2BUILD)" emconfigure \
```

Comment:
Why do we need this? Wouldn't that mean compiling to asm.js instead of WASM?
https://github.com/emscripten-core/emscripten/blob/0ea89abddbc5523a5d4912cac0dc95be3699068e/src/settings.js#L1064

Reply:
Yes. I've written a bit about it in the commit message. Now that I think about it, perhaps the right solution would be to enable EMULATE_FUNCTION_POINTER_CASTS here. Let me try that

Reply:
Setting WASM=0 will also probably be faster, and `./configure` takes up a good chunk of the compilation time.

Reply:
OK, I saw commit message in https://github.com/iodide-project/pyodide/pull/843/commits/61342c2832f5e58da8fa6da711799415b9a4f789 sounds good to me then.

Reply:
EMULATE_FUNCTION_POINTER_CASTS doesn't actually work